### PR TITLE
Coral-Visualization: Add Coral visualization json tests

### PR DESCRIPTION
**Pull request by wmoustafa**
### What changes are proposed in this pull request, and why are they necessary?
This PR adds Json based tests to the Coral visualization module, so that basic checks can be performed on the output visualization. In the current tests, the existence of certain expected node labels is checked.

This patch also adds a lateral view test query.

### How was this patch tested?
`./gradlew build`

### CODE REVIEW

File: coral-visualization/src/test/java/com/linkedin/coral/vis/VisualizationUtilTest.java [hunk 1]
```diff
@@ -18,27 +26,103 @@
...
 import static org.testng.Assert.*;
 
 
 public class VisualizationUtilTest {
   private static final String CORAL_VISUALIZATION_TEST_DIR = "coral.visualization.test.dir";
 
   @Test
-  public void testBasicQuery() {
+  public void testRenderToFile() {
+    String[] queries =
```
Comment (aastha25) [inline coral-visualization/src/test/java/com/linkedin/coral/vis/VisualizationUtilTest.java:44]: Can you extract out the SQLs to test in a data provider like [here](https://github.com/linkedin/coral/blob/711268b04cff8057d7327f1c72c8d516461c006d/coral-trino/src/test/java/com/linkedin/coral/trino/rel2trino/HiveToTrinoConverterTest.java#L51) ?

**Review by aastha25 — COMMENTED** [review]
LGTM! just left a minor nit comment.

File: coral-visualization/src/test/java/com/linkedin/coral/vis/VisualizationUtilTest.java (hunk 1 above)
Comment (wmoustafa) [reply coral-visualization/src/test/java/com/linkedin/coral/vis/VisualizationUtilTest.java:44]: I do not expect this list to grow much. This is just to test the functionality of writing the image to a file. Individual tests will have their own test cases.

**Review by yiqiangin — APPROVED** [review]

File: coral-visualization/src/test/java/com/linkedin/coral/vis/VisualizationUtilTest.java [hunk 2]
```diff
@@ -18,27 +26,103 @@
...
+    return true;
+  }
+
+  private static boolean jsonKeyValueExists(Object object, String key, String value) {
+    if (object instanceof JsonObject) {
+      JsonObject jsonObject = (JsonObject) object;
+      if (jsonObject.has(key)) {
+        if (jsonObject.get(key).getAsString().equals(value)) {
+          return true;
+        }
```
Comment (ljfgem) [inline coral-visualization/src/test/java/com/linkedin/coral/vis/VisualizationUtilTest.java:107]: nit: can be simplified to be `return jsonObject.get(key).getAsString().equals(value);`

**Review by ljfgem — APPROVED** [review]
LGTM

**Pull request merged**